CHANGELOG.md

```diff
@@ -2,12 +2,57 @@
 
 ## Unreleased
 
+## v0.116.0
+
+This Splunk OpenTelemetry Collector release includes changes from the opentelemetry-collector v0.116.0 and the opentelemetry-collector-contrib v0.116.0 releases where appropriate.
```

Comment:
```suggestion
This Splunk OpenTelemetry Collector release includes changes from the [opentelemetry-collector v0.116.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.116.0) and the [opentelemetry-collector-contrib v0.116.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.116.0) releases where appropriate.
```

I see there's a contrib release `v0.116.1`, out of ignorance, should we be basing our release off of that release version, or `v0.116.0`?

Reply:
the releases job failed and the team had to make a new release binary with a new version, but there's no contrib changes.